backend/api/serializers.py

```diff
@@ -0,0 +1,278 @@
+    own_attendance_status = serializers.SerializerMethodField()
+    own_follow_status = serializers.SerializerMethodField()
+    own_vote = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Event
+        fields = ('id', 'owner', 'title', 'description', 'date', 'price',
+                  'created', 'updated', 'own_attendance_status', 'follower_count',
+                  'own_follow_status', 'vote_count', 'own_vote')
+
+    def get_own_attendance_status(self, obj):
+        user = self.context.get("request").user
+        if user and user.is_authenticated:
+            own_attendance = obj.attendance_status.all().filter(owner=user).first()
+            return AttendanceDetailsSerializer(own_attendance).data if own_attendance else None
+        return None
+
+    def get_own_follow_status(self, obj):
+        user = self.context.get("request").user
+        if user and user.is_authenticated:
+            own_follow = obj.followers.all().filter(owner=user).first()
+            return FollowDetailsSerializer(own_follow).data if own_follow else None
+        return None
+
+    def get_own_vote(self, obj):
+        user = self.context.get("request").user
+        if user and user.is_authenticated:
+            own_vote = obj.votes.all().filter(owner=user).first()
+            return VoteDetailsSerializer(own_vote).data if own_vote else None
+        return None
+
+
+class EventCreateUpdateSerializer(serializers.ModelSerializer):
+    medias = MediaDependentCreateSerializer(many=True)
+
+    class Meta:
+        model = Event
+        fields = ('id', 'title', 'description', 'date', 'price',
+                  'organizer_url', 'artists', 'medias', 'tags')
+
+    def create(self, validated_data):
+        owner = self.context.get("request").user
+        artists_data = validated_data.pop('artists', [])
+        medias_data = validated_data.pop('medias', [])
+        tags_data = validated_data.pop('tags', [])
+        event = Event.objects.create(owner=owner, **validated_data)
+        for artist in artists_data:
+            event.artists.add(artist)
+        for media_data in medias_data:
+            Media.objects.create(owner=owner, content_object=event, **media_data)
+        for tag in tags_data:
+            event.tags.add(tag)
+        return event
+
+    def update(self, instance, validated_data):
+        owner = self.context.get("request").user
+        # If 'artists' key is given in data, clear current artists
+        # and add new ones. Else, discard it.
+        artists_data = None
+        try:
+            artists_data = validated_data.pop('artists')
+            instance.artists.clear()
+            for artist in artists_data:
+                instance.artists.add(artist)
+        except KeyError:
+            pass
+
+        # If 'medias' key is given in data, delete current medias
+        # and add new ones. Else, discard it.
+        medias_data = None
+        try:
```

Comment:
the same with the above comment. you can apply this all `try-catch`es in this function